### ISSUE TITLE
Feature Request: Widget `compute`

### ISSUE THREAD

**Issue by samtrion** [sveltia/sveltia-cms#111]
This widget is a read-only element that updates itself based on events (focus change, etc.). It should take over existing content and transform it if necessary.

The configuration for this could look as follows.
```yml
fields:
  - label: Slug
    name: slug
    widget: string
    i18n: duplicate
  - label: TranslationKey
    name: translationKey
    widget: compute
    value: posts-{{fields.slug}}
```

**Comment by kyoshino** [comment]
It’s probably not difficult to implement but I don’t want to think about circular reference handling 😅

**Comment by samtrion** [comment]
Well, there are solutions for that too, especially as the range of setting options is limited.

It also depends a little on the implementation of #17, so that you don't have to fish with dynamite to find the error.

**Comment by kyoshino** [comment]
Just shipped [v0.13.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.13.0) to implement this and other features 🚀
